text/XXX-A-Compilation-of-Short-Interactive-Tutorials-and-How-To-Guides-for-Coq.md

```diff
@@ -0,0 +1,257 @@
+
+Compared to the reference manual, such a documentation is not meant to be complete or going into low level details. It is meant to be pedagogical and action-oriented, so that it enables users to learn how to use new features autonomously and horizontally : these tutorials and how-to guides should be as self-contained as possible to have only a short (if any) list of prerequisites. Moreover, by being horizontal, such a documentation could provide different tutorials (for some specific features) to cover the needs of users with different backgrounds.
+
+---
+## 2. Motivation
+
+Having a proper, clean and accessible documentation is one of the keys to the success of software. There are different forms of documentation: abstract and detailed documentation like the reference manual, course-shaped documentation like Coq'Art or Software Foundations, or short action-oriented documentation. In this CEP, we focus on the latter.
+
+
+### 2.1 The lack of action-oriented documentation
+Short action-oriented documentation provides users with practical information on specific features of Coq, so that they can both learn and discover new features by themselves, but also consult them when they are trying to use these features and are stuck.
+
+Apart from long, course-shaped books, Coq currently has only minimal and scattered action-oriented documentation.
+The reference manual is by design not learning-oriented and not action-oriented, and it would be a mistake to try to bend it that way: it is not its purpose. It also provides relatively few examples (although many new examples have been added in the past few years and more would be welcome) and it is focused on Coq core features.
+Books like Coq'Art or Software Foundations provide nice pedagogical explanations but target specific audiences, and are often meant to be read from cover to cover. Moreover, they are not well suited to learn about specific features, to discover horizontally, and are not easy to keep updated.
+Other great resources exist like [Coq Tricks](https://github.com/tchajed/coq-tricks) or package-specific tutorials, but they are limited and are not necessarily easy to access for newcomers.
+
+
+### 2.2 Why it is important
+Not having a central collection of short and action-oriented documentation pieces is a real obstacle to getting more users outside of expert academic fields, and for Coq's popularity.
```

Comment:
Fixing an exact length would not be useful because it really depends on the topic and on the density of the content. Yet, for indication, so far, we have kept all tutorials below 1000 lines of code and that seems reasonable. 

But to me, a better way to decide is to:
- Check the structure of the content. If it starts branching out in different directions,
- Also when the writer or the reviewer have trouble getting around the code and the concepts, it's a good indication it is too long.